### DISCUSSION TITLE
Languages

### DISCUSSION THREAD

**Discussion by jorgesenameh** [sanidhyy/duolingo-clone#19] [Q&A] [answered]
The app has Spanish language, we are Spanish speakers who want to practice English, we need the interface in Spanish.

Thank you, we look forward to your prompt response

**Comment by sanidhyy** [comment, marked as answer]
Thank you for your question.

If you're interested in adding support for multiple languages, I recommend reviewing the Next.js documentation on internationalization: https://nextjs.org/docs/app/building-your-application/routing/internationalization.

**Reply by jorgesenameh** [reply]
Hi, You can share a login access, when trying to create a user it is not possible because the limit of 500 users has been reached.

Thank you

**Reply by sanidhyy** [reply]
Hi @jorgesenameh,

I’ve created a new test user account for you. Please use the following credentials to log in:

**Email:** test@gmail.com
**Password:** password

Let me know if you need anything else.

Thank you!

**Comment by jorgesenameh** [comment]
Hi, I've already installed Lingo on my server, www.bmtalk.co. Is it possible for you to provide us with support to have the platform in Spanish for people who want to learn English? How much would it cost?

Thanks,
Jorge

**Comment by sanidhyy** [comment]
Hi @jorgesenameh,

Please **contact me** directly at [my email](mailto:sanidhya.verma12345@gmail.com) so we can discuss the details.

Best,
**Sanidhya**
